python-backend/wolfram_client.py

```diff
@@ -73,40 +59,14 @@ def closed_form(expression: str) -> dict:
         :return: Wolfram API response "subpods" with the computed limit(s)
         """
         try:
-            result = WolframClient.ask(query=expression, include_pod="PossibleClosedForm")
+            # Wolfram has a 200 character input limit
+            result = WolframClient.ask(query=expression[:200], include_pod="PossibleClosedForm")
```

Comment:
Nit: Rather than truncating the length of the expression, I'd recommend asserting that the expression is under 200 characters (or maybe logging an error). I can imagine subtle and unexpected issues happening for complicated queries that are longer than 200 characters that may be difficult to diagnose unless there is something to tell them what is happening under the hood.